bennu-core/src/main/java/org/fenixedu/bennu/core/api/json/UserJsonAdapter.java

```diff
@@ -101,12 +101,17 @@ private void changePassword(User user, JsonObject json) {
         String password = JsonUtils.getString(json, "password");
         if (Strings.isNullOrEmpty(password)) {
             // Dont' change the password
-            return;
+            throw BennuCoreDomainException.passwordIsEmpty();
```

Comment:
Won't this change prevent the use of the User Management interface without inputting the password? (For when information other than the password is being edited).